src/main/webapp/file.xhtml

```diff
@@ -552,9 +553,9 @@
                                                         <td>#{FilePage.file.publicationDateFormattedYYYYMMDD}
                                                         </td>
                                                     </tr>
-                                                    <tr id="fileEPublicationDateBlock" jsf:rendered="#{FilePage.file.released or !(empty FilePage.file.embargo)}">
+                                                    <tr id="fileEPublicationDateBlock" jsf:rendered="#{!(empty FilePage.file.embargo)}">
```

Comment:
Removed because the Embargo End date was also rendered when there was no embargo, but it was published (released)

Reply:
I don't understand this change. The Publication Date is supposed to show in both cases. It will either be the real publication date or will show the embargo end date if it is set.

Reply:
The publication date is shown above in the element with `id="metadataReleaseDateBlock"`. Also the element with `id="fileEPublicationDateBlock"` has the label that states it is about embargo, so why should it be shown if there is no embargo. 
When I tested the retention period I got both retention and embargo elements, just showing the publication date, which was weird. Therefore I changed it, but I wonder why I was the only one seeing it.

Reply:
It looks like you've changed the label in the line 558 below - it would normally say publication date. I think the design guidance was to always show the metadata release date and the publication date all the time. With no embargo, those dates are the same. With an embargo, the pub date is delayed.

Reply:
Just changed it, but the composition of all those options in one element makes it a bit complicated; al lot of those `#{}` combined.